### ISSUE TITLE
Add Temporal Planner

### ISSUE THREAD

**Issue by haz** [AI-Planning/planning-as-a-service#8]
Mirror of [this issue](https://github.com/AI-Planning/cloud-solver/issues/5).

**Comment by jan-dolejsi** [comment]
Done already? Two temporal planners are available now. No convention for the plan format though.

**Comment by haz** [comment]
Yes indeed! And conventions are intentionally chaotic -- more to say on another issue...
